pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -116,7 +116,8 @@ func Convert_v1_Disk_To_api_Disk(diskDevice *v1.Disk, disk *Disk, devicePerBus m
 func checkDirectIOFlag(path string) bool {
 	// check if fs where disk.img file is located or block device
 	// support direct i/o
-	f, err := os.OpenFile(path, syscall.O_RDONLY|syscall.O_DIRECT, 0)
+	f, err := os.OpenFile(path, syscall.O_RDONLY, 0)
+	//f, err := os.OpenFile(path, syscall.O_RDONLY|syscall.O_DIRECT, 0)
```

Comment:
Why do you change it?
I also think some CI failures can be related to this change
```
Configurations with driver cache settings [It] should set appropriate cache modes
```

Reply:
@cynepco3hahue good catch, the O_DIRECT flag must be here.

Reply:
Oh thank you! What a stupid mistake on my side.
It seems like there is no syscall.O_DIRECT on macOS. I therefore removed the line to prevent an error on my laptop and then I forgot to put it back.

Reply:
So at least this needs to be fixed.